src/controller.ts

```diff
@@ -66,6 +66,8 @@ function *ParserController () {
                     keys = response.keys
                 }
             }
+            if (ex_str != "repeat")
```

Comment:
This is still vulnerable to recursion errors if the last ex_str was e.g. `repeat 5`.

Probably best to make the decision in acceptExCmd after the ex_str has been parsed but before it is executed.

Reply:
You're right, this should have been ex_str.startsWith("repeat").

Moving this check to acceptExCmd is what I did at first, however it makes Tridactyl behave differently from Pentadactyl: in Pentadactyl, dot-repeat after pressing 't' opens the commandline, this is also Tridactyl's behavior with the current code.
Moving the code to acceptExCmd would make Tridactyl directly open a new tab, while that makes more sense to me it might confuse people expecting Pentadactyl's behavior.

Let me know what you prefer, I don't use dot-repeat enough for this change to matter to me.